test/pathBuilder.test.js

```diff
@@ -71,15 +71,15 @@ describe('documentation paths', () => {
     });
 
     it('css distribution directory path should be correct', () => {
-        expect(pathBuilder.docsCssDistDir).toBe('./docs/dist/assets/css');
+        expect(pathBuilder.docsCssDistDir).toBe('./docs/dist/assets/dist/css');
```

Comment:
Did the scss image `url`s not work before then?  Or is this just a new problem with the way we’ve setup the new tasks